modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/confirm.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export function confirm({message, onConfirm}) {
+	Liferay.Util.openModal({
+		bodyHTML: message,
+		buttons: [
+			{
+				displayType: 'secondary',
+				label: Liferay.Language.get('cancel'),
+				type: 'cancel',
+			},
+			{
+				label: Liferay.Language.get('ok'),
+				onClick: ({processClose}) => {
+					processClose();
+
+					onConfirm(true);
+				},
+				otherProps: {
+					autoFocus: true,
```

Comment:
This `autoFocus` was necessary to reach the same original behavior by focusing the Ok button and it was a "fix" for trapping screen reader to read the content inside the modal as it does on the original implementation.

Another possibility to implement this trap would be a React component like this [FocusTrap from MUI](https://mui.com/components/trap-focus/) where we can make sure the opened Modals will not make the screen reader lost by reading the elements behind Modal stuff. /cc @matuzalemsteles @julien

Reply:
We can study that, but I think it would be better to add this feature straight to ClayModal I don't know what the other use cases would be for this, maybe I'm not seeing it right now.

Reply:
Should I create an issue for this?

Reply:
Yes please. I think this would be a nice accessibility improvement for the component.